packages/themed/src/components/Textarea/styled-components/Input.tsx

```diff
@@ -5,8 +5,6 @@ export default styled(
   TextInput,
   {
     p: '$2',
-    //@ts-ignore
-    multiline: true,
```

Comment:
Hey @njt1982, `multiline` is a part of a prop and not the styling that's why we were getting this issue.
moving it inside props that should fix it.
```
props: {
      placeholderTextColor: '$textLight500',
      multiline: true,
    },
```

Reply:
I’ll make that change. Thanks!

Reply:
@Viraj-10 done - I've left the `//@ts-ignore` comment, not sure if its still needed.

I've manually applied this change to my local node_modules checkout for my project and it seems happy enough...

Reply:
Awesome 🚀 .